### ISSUE TITLE
cannot hide an action_group of links

### ISSUE THREAD

**Issue by m0x** [vhochstein/active_scaffold_batch#2]
seems more like a pull request, but since i'm übernoob, my problem may just be caused by stupidity, so ... let's just say it's an issue.
when you use a security_method to hide batch links, the <div class="batch"> container is still generated with empty content (ugly). I've been skimming through _action_group.html.erb & action_links.rb in your as fork, and i could not find any way to hide a whole action group. Is it possible ? 

By the way, your fork is awesome.

**Comment by vhochstein** [comment]
Yes, that s true that s an issue, but a general issue of activescaffold and not just activescaffoldbatch.

I ve seen that issue already in my apps, but I have nt developed a smart way to fix this, cause it s quite a rare case and I do not think it would be good practice to check all ignore and security methods everytime from a performance view just to fix this rare case...

**Comment by m0x** [comment]
ok, thanks for the reply. I just used a (ugly) hack to fix this, which consists in a conditionnal style element in my layout:

<% if can_see_links? %>
  <style>div.batch{display:none}</style>
<% end %>

performance is not an issue as my site won't see a lot of traffic.

**Comment by vtt** [comment]
You can hide a whole action group by removing it.  Use a before filter (see https://github.com/activescaffold/active_scaffold/wiki/Per-Request-Configuration) and add "active_scaffold_config.action_links.collection.delete_group :batch"
